docs/containers/quickstart-node.md

```diff
@@ -85,21 +85,9 @@ The extension will create `Dockerfile` and `.dockerignore` files. If you elected
 
 When the Docker extension adds files to the application, it also adds a **VS Code debugger configuration** in `.vscode/launch.json` for debugging the service when running inside a container. The extension detects the protocol and port used by the service and points the browser to the service, but make sure the application's output is written to the debug console.
 
```

Comment:
The "...but make sure the application's output is written to the debug console" clause can be omitted as well.